### PR TITLE
Cleanup MsgWaitForMultipleObjectsEx

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Cleanup MsgWaitForMultipleObjectsEx
- Cleanup QS_ constants
- Cleanup MWM_ constants

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2042)

### CODE REVIEW

File: src/Common/src/Interop/User32/Interop.MWMO.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum MWMO : uint
```
Comment (weltkante) [inline src/Common/src/Interop/User32/Interop.MWMO.cs]: Shouldn't this have a `[Flags]` attribute?

```suggestion
        [Flags]
        public enum MWMO : uint
```

File: src/Common/src/Interop/User32/Interop.QS.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum QS : uint
```
Comment (weltkante) [inline src/Common/src/Interop/User32/Interop.QS.cs]: Shouldn't this have a `[Flags]` attribute?

```suggestion
        [Flags]
        public enum QS : uint
```

File: src/Common/src/Interop/User32/Interop.QS.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        {
+            KEY = 0x0001,
+            MOUSEMOVE = 0x0002,
+            MOUSEBUTTON = 0x0004,
+            POSTMESSAGE = 0x0008,
+            TIMER = 0x0010,
+            PAINT = 0x0020,
+            SENDMESSAGE = 0x0040,
+            HOTKEY = 0x0080,
+            ALLPOSTMESSAGE = 0x0100,
```
Comment (zsd4yr) [inline src/Common/src/Interop/User32/Interop.QS.cs]: Missing come values,
QS_RAWINTPUT 0x0400

and then QS_INPUT is actually an or of QS_MOUSE, QS_KEY, and QS_RAWINPUT

Comment (hughbe) [reply src/Common/src/Interop/User32/Interop.QS.cs]: Updated thanks for spotting

**Pull request merged**